qutebrowser/config/configtypes.py

```diff
@@ -1159,8 +1148,8 @@ class FontBase(BaseType):
     """Base class for Font/FontFamily."""
 
     # Gets set when the config is initialized.
-    default_family = None  # type: str
-    default_size = None  # type: str
+    default_family: Optional[str] = None
+    default_size: Optional[str] = None
```

Comment:
I had to change these to `Optional` to allow the setting of `None`. From my reading of the code this looked like the correct thing to do (i.e. there are falsey/truthy checks) rather than defining the type without a default value.